### DISCUSSION TITLE
Presets for Aufero Laser Engraver 10w optical?

### DISCUSSION THREAD

**Discussion by johnsebooks** [arkypita/LaserGRBL#2244] [General]
I just installed a 10W LU2-10A Laser Module Aufero Laser Engraver on my Ortur 2 S2 and LRBL's presets don't list it. There a setting for Ortur's lower-wattage lasers but not this one. Can I  use a different brands 10W settings? Any help?

**Comment by Tribo-2dot0** [comment]
São apenas referências, use um outro de 10W semelhante ao seu e inicie por aí... as últimas linhas em branco você pode ir preenchendo com seus próprios parâmetros.

**Reply by johnsebooks** [reply]
Obrigado, vou começar a procurar um sistema semelhante e brincar com suas configurações.
